react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class TopicCard extends React.Component {
+  render() {
+    let visibility;
+    this.props.clickedTopicID === this.props.id && this.props.clickCount < 1 ? visibility = 'show' : visibility = 'hide';
```

Comment:
A better way to write this would be:

```JSX
let visibility = this.props.clickedTopicID === this.props.id && this.props.clickCount < 1
  ? 'show'
  : 'hide';
```